bin/mv.js

```diff
@@ -0,0 +1,239 @@
+'use strict';
+
+const url = require('url');
+const _ = require('lodash');
+const coroutine = require('bluebird').coroutine;
+const debug = require('debug')('wt-cli:mv');
+const request = require('superagent');
+const chalk = require('chalk');
+const Cli = require('structured-cli');
+const ConfigFile = require('../lib/config');
+
+module.exports = Cli.createCommand('mv', {
+    description: 'Move a named webtask',
+    plugins: [
+        require('./_plugins/profile')
+    ],
+    options: {
+        'target-container': {
```

Comment:
If you allow target container to be specified as an override of target-profile, you should probably also allow an override of target-url and target-token for completeness.